theme/sass/components/_global.scss

```diff
@@ -43,4 +43,19 @@ body {
 .responsive-image {
     width: 100%;
     height: auto;
-}
\ No newline at end of file
+}
+
+.clearfix:after {
+    content: " ";
+    visibility: hidden;
+    display: block;
+    height: 0;
+    clear: both;
+}
+
+h1 {
+    font-size: $font-large;
```

Comment:
Perhaps all H1's on the site should be x-large so that they look visually different from the H2's

Reply:
done